src/screens/Followings/elements/TabBar/FollowingTabBar.js

```diff
@@ -56,8 +56,8 @@ const MyTabBar = ({state, descriptors, position, navigation}) => {
 const S = StyleSheet.create({
   toptabcontainer: {
     flexDirection: 'row',
-    backgroundColor: colors.white,
-    borderBottomColor: '#00000050',
+    backgroundColor: COLORS.white,
+    borderBottomColor: COLORS.black30percent,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hard-coded color value `'#00000050'` has been replaced with a reference to a color constant `COLORS.black30percent`. This change enhances maintainability and consistency across the codebase.
```diff
-    backgroundColor: colors.white,
-    borderBottomColor: '#00000050',
+    backgroundColor: COLORS.white,
+    borderBottomColor: COLORS.black30percent,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->